zerver/openapi/zulip.yaml

```diff
@@ -5895,6 +5895,51 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/JsonSuccess"
+    patch:
+      operationId: update_linkifier
+      tags: ["server_and_organizations"]
+      description: |
+        Update [linkifiers](/help/add-a-custom-linkification-filter), regular
+        expression patterns that are automatically linkified when they appear
+        in messages and topics.
```

Comment:
This should make it more clear that this is updating a single linkifier.

Reply:
Done